dstep/Configuration.d

```diff
@@ -77,6 +77,10 @@ struct Configuration
     /// do not translate following symbols
     @("skip", "Skip translation of <symbol>.")
     string[] skipSymbols;
+
+    /// enable printing of warnings
+    @("enable-warnings", "Enable printing of warnings [default].")
+    bool enableWarnings = true;
```

Comment:
I added two new options: `--print-diagnostics` and `--collision-action`. A collision action is an enum, so I needed to change the help to accommodate for that. Here is the new help:

```
Usage: dstep [options] <input>
Version: v0.2.2-112-gf88c046

Options:
    -o, --output <file>                           Write output to <file>.
    -o, --output <directory>                      Write all the files to <directory>, in case of multiple input files.
    -ObjC, --objective-c                          Treat source input file as Objective-C input.
    -x, --language                                Treat subsequent input files as having type <language>.
    --version                                     Show dstep version.
    --clang-version                               Show libclang version.
    --package <package>                           Use <package> as package name.
    --comments=true|false                         Translate comments [default: true].
    --public-submodules=true|false                Use public imports for submodules [default: false].
    --reduce-aliases=true|false                   Reduce primitive type aliases [default: true].
    --portable-wchar_t=true|false                 Translate wchar_t as core.stdc.stddef.wchar_t [default: true].
    --zero-param-is-vararg=true|false             Translate functions with no arguments as variadic functions [default: false].
    --single-line-function-signatures=true|false  Keep function signatures in a single line [default: false].
    --space-after-function-name=true|false        Put a space after a function name [default: true].
    --skip-definition <symbol>                    Keep only the signature of <symbol>.
    --skip <symbol>                               Skip translation of <symbol>.
    --print-diagnostics=true|false                Print diagnostic informations [default: true].
    --collision-action=ignore|rename|abort        Action to take when translated symbol collides with a preexisting symbol [default: rename].
    -h, --help                                    This help information.

All options that Clang accepts can be used as well.
Use the `-h' flag for help.
```

Tell me what do you think.